code/modules/mob/living/carbon/human/species.dm

```diff
@@ -174,6 +176,15 @@
 			tongue = new mutanttongue
 			tongue.Insert(C)
 
+		if(brain && brain.type != mutant_brain && !brain.decoy_override)
+			var/mob/dead/observer/ghost = C.ghostize(0)
+			qdel(brain)
+			brain = new mutant_brain()
+			brain.Insert(C)
+			ghost.mind.transfer_to(C)
```

Comment:
why does this feel like the hackiest thing I've seen in a while. @AnturK

Reply:
I could just transfer brain to new mob but they'd get the disoriented message on species change